cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -325,21 +358,24 @@ impl cumulus_pallet_parachain_system::Config for Runtime {
 impl parachain_info::Config for Runtime {}
 
 parameter_types! {
-	pub MessageQueueServiceWeight: Weight = Perbill::from_percent(35) * RuntimeBlockWeights::get().max_block;
+	/// Amount of weight that can be spent per block to service messages.
+	pub MessageQueueServiceWeight: Weight = Perbill::from_percent(60) * RuntimeBlockWeights::get().max_block;
```

Comment:
@svyatonik we need to import the beacon checkpoint at the start of the Ethereum light client cycle. The checkpoint payload contains the public keys of the initial sync committee validator set - 512 keys of 48 bytes each. That's what prompted this change. Here's the extrinsic: https://github.com/Snowfork/snowbridge/blob/main/parachain/pallets/ethereum-beacon-client/src/lib.rs#L221

Reply:
Got it, thank you! Then imo it worth to do one of following (or both):
1) add some docs here, explaining that this `MessageQueueServiceWeight` we need to cover all (?) beacon pallet weights (or at least `force_checkpoint`);
2) add a test for that that would fail if `force_checkpoint` weight is smaller than the `MessageQueue::max_message_weight()`